make-release.sh

```diff
@@ -168,6 +173,7 @@ if [[ ${PHASES} == *"1"* ]]; then
     releaseMachineExec
     releaseDashboard
```

Comment:
above you moved dashboard to phase 2; here it's still phase 1. I'm confused. Please clarify or at least be consistent with your proposed changes. :D